lib/sqlitex/server/statement_cache.ex

```diff
@@ -34,7 +34,7 @@ defmodule Sqlitex.Server.StatementCache do
     end
   end
 
-  defp prepare_new_statement(%__MODULE__{db: db} = cache, sql, opts \\ []) do
+  defp prepare_new_statement(%__MODULE__{db: db} = cache, sql, opts) do
```

Comment:
dialyzer under elixir `1.4.5` was also throwing an error about this because there was no codepath that could call this function without the 3rd argument. So the optional argument was never used, so I simplified it to just be an expected argument.